other/materials_designer/create_perturbation_custom.ipynb

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
<p>function -&gt; custom_perturbation_function_sympy_xy</p><p> </p><p>Should be isolated to its own cell, as the rest is equivalent between this one and the other notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>